pypeit/calibrations.py

```diff
@@ -562,10 +576,22 @@ def get_flats(self):
 
         # Only build illum_flat if the input files are different from the pixel flat
         if not pix_is_illum and len(illum_image_files) > 0:
+            msgs.info('Creating Master Illumination Flat using files: ')
+            for f in illum_image_files:
+                msgs.info(f'{os.path.basename(f)}')
             illum_flat = buildimage.buildimage_fromlist(self.spectrograph, self.det,
                                                         self.par['illumflatframe'],
                                                         illum_image_files, dark=self.msdark,
                                                         bias=self.msbias, bpm=self.msbpm)
+            if len(flatLoff_image_files) > 0:
```

Comment:
I think requires a bit more thought. Suppose the user uses sky spectra as illumination flats, which is sometimes done. Then I don't want to subtract the lampoff flat from the sky spectrum. Again, another case where we need a unique association algorithm. Anyway, I think what you are doing only makes sense for the case where the dome flat and illumflat come from the same set of flats, i.e. if pix_is_illum is True.

Reply:
Added a note in the documentation.